bezpy/mt/io.py

```diff
@@ -189,3 +192,29 @@ def get_1d_site(name):
         return _SITES1D[newname]
 
     raise ValueError("No 1d site profile with the name: " + name)
+
+def read_runinfo(site,root):
+    site.runinfo = {}
+
+    # fieldnotes = xml_site.find("FieldNotes")
+    for field in root.findall("FieldNotes"):
+        # runid of fieldnote
+        runid = field.attrib['run']
+        site.runinfo[runid] = {}
+
+        try:
+            site.NIMSid = get_text( field.find('Instrument'), 'Id')
+        except KeyError:
+            site.NIMSid = None
+
+        # run through E component
+        for ecomp in field.findall("Dipole"):
+            # component
+            Edirection = ecomp.attrib['name']   # Ex or Ey
+            site.runinfo[runid][Edirection] = convert_float(get_text(ecomp, "Length"))
+
+        # set start and end datetime
+        site.runinfo[runid]['Start'] = datetime.datetime.strptime( field.find('Start').text,'%Y-%m-%dT%H:%M:%S')
```

Comment:
There is extra white space after the parenthesis in several places. There is also missing whitespace after the comma here.

Reply:
Thanks. I've updated those white spaces using flake8 and pylint